packages/react/test/specs/components/Attachment/Attachment-test.tsx

```diff
@@ -18,5 +21,32 @@ describe('Attachment', () => {
     handlesAccessibility(Attachment, {
       defaultRootRole: undefined,
     })
+
+    test('handleClick is executed when Enter is pressed for attachment, not for more option button', () => {
+      const onClickAttachment = jest.fn()
+      const onClickButton = jest.fn()
+      const attachement = mountWithProvider(
+        <Attachment
+          actionable
+          action={{
+            icon: 'more',
+            onClick: () => onClickButton(),
+          }}
+          onClick={onClickAttachment}
+        />,
+      )
+      const moreOptionButton = attachement.find('button')
+
+      attachement.simulate('keydown', { keyCode: keyboardKey.Enter })
+      expect(onClickAttachment).toHaveBeenCalledTimes(1)
+
+      onClickAttachment.mockClear()
+      moreOptionButton.simulate('keydown', { keyCode: keyboardKey.Enter })
+      expect(onClickAttachment).not.toBeCalled()
+
+      // simulate the similar behavior as browser does when Enter key is pressed on button element
+      moreOptionButton.simulate('click')
+      expect(onClickButton).toHaveBeenCalledTimes(1)
```

Comment:
I think this part should be tested separately, if not already tested in `isConformant`. Maybe you can check with someone from Stardust whether this handler call is not already tested.